artview/components/roi.py

```diff
@@ -192,16 +192,30 @@ def CreateROIWidget(self):
         self.buttonOpenTable = QtGui.QPushButton('Open Tabular Data', self)
         self.buttonSaveTable = QtGui.QPushButton('Save Tabular Data', self)
         self.buttonResetROI = QtGui.QPushButton('Reset ROI', self)
+        self.buttonHelp = QtGui.QPushButton('Help', self)
         self.buttonViewTable.clicked.connect(self.viewTable)
         self.buttonOpenTable.clicked.connect(self.openTable)
         self.buttonSaveTable.clicked.connect(self.saveTable)
         self.buttonResetROI.clicked.connect(self.resetROI)
+        self.buttonHelp.clicked.connect(self.displayHelp)
 
         # Create functionality buttons
+
         self.rBox_layout.addWidget(self.buttonViewTable)
         self.rBox_layout.addWidget(self.buttonOpenTable)
         self.rBox_layout.addWidget(self.buttonSaveTable)
         self.rBox_layout.addWidget(self.buttonResetROI)
+        self.rBox_layout.addWidget(self.buttonHelp)
+
+    def displayHelp(self):
+        # XXX I invite anyone to improve this help information
+        text = "Draw a Path in the Display Window using the Mouse\n\n"
+        text += "Functions:\n"
+        text += "    Primary Mouse Button - add vertex\n"
+        text += "    Hold - freely draw path\n"
+        text += "    Secundary Button - close path\n"
```

Comment:
I could go either way on this one as well. I could certainly see the utility in just putting this string in the doc-string and pulling it in from there. Only issue might be if the doc-string continues to grow in the future...hmmm